tests/testall.sh

```diff
@@ -1,49 +0,0 @@
-#!/bin/sh
```

Comment:
this file runs tests locally, for developer's convenience; but the environment file example is not in the REPO. Removing it as as far as I know no one but me uses it.